imu_filter_madgwick/src/imu_filter.cpp

```diff
@@ -51,6 +51,10 @@ ImuFilter::ImuFilter(ros::NodeHandle nh, ros::NodeHandle nh_private):
   if (!nh_private_.getParam ("publish_debug_topics", publish_debug_topics_))
     publish_debug_topics_= false;
 
+  double orientation_stddev;
+  nh_private_.param<double>("orientation_stddev", orientation_stddev, 0.0);
+  orientation_variance_ = orientation_stddev * orientation_stddev;
+
```

Comment:
You don't need lines 54-57. It's automatically taken care of by dynamic reconfigure. What happens is:

* we register the dynamic reconfigure server (L75)
* it reads the parameters and calls `reconfigCallback()`

This happens before we initialize the subscribers, so we can be sure that the param has been processed before we first access `orientation_variance_`.